frontend/packages/ceph-storage-plugin/integration-tests/views/subscriptionFlow.view.ts

```diff
@@ -0,0 +1,199 @@
+import { execSync } from 'child_process';
+import { $, ExpectedConditions as until, browser, element, by } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import * as _ from 'lodash';
+
+// TODO => Captialize S in OpenShift
+
+export const OCS_OP = 'Openshift Container Storage Operator';
+
+export const NS = 'openshift-storage';
+
+export const operatorsPage = async () => {
+  await sideNavView.clickNavLink(['Operators', 'Installed Operators']);
+  await browser.wait(until.and(crudView.untilNoLoadersPresent));
+};
+
+export const ocsOperator = $('.co-clusterserviceversion-logo__name__clusterserviceversion');
+export const ocsOperatorStatus = $('.co-clusterserviceversion-row__status');
+
+const ocsLink = (elem, catalogSource) =>
+  $(`a[data-test="${elem}-${catalogSource}-openshift-marketplace"]`);
+
+export const goToOperatorHub = async () => {
+  await sideNavView.clickNavLink(['Operators', 'OperatorHub']);
+  await browser.wait(until.and(crudView.untilNoLoadersPresent));
+};
+
+const searchInputOperatorHub = $('input[placeholder="Filter by keyword..."]');
+
+export const searchInOperatorHub = async (searchParam) => {
+  await browser.wait(until.visibilityOf(searchInputOperatorHub));
+  await searchInputOperatorHub.sendKeys(searchParam);
+  const ocs = await ocsLink('ocs-operator', 'ocs-catalogsource');
```

Comment:
Move this to a variable. This can change for various reasons.

Reply:
can't change. 4.2 CSS is frozen. I don't think we should move all id, class values to a separate constant.

Reply:
This change comes from backend. Which will change the day QE starts testing from release repo :)

Reply:
but even if we move it to a variable how are we going to pass the value at runtime without modifying the code?

Reply:
Environment Variables or Function Arguments are a foreign concept in TS? I assume that would be the right way to pass this value. Even without it, using variable/const is much preferable.

Reply:
env vars are available in JS.